KNL/events/KnightsLanding_core.json

```diff
@@ -891,7 +891,7 @@
       "EventCode": "0x86",
       "UMask": "0x04",
       "EventName": "FETCH_STALL.ICACHE_FILL_PENDING_CYCLES",
-      "BriefDescription": "Counts the number of core cycles the fetch stalls because of an icache miss. This is a cummulative count of core cycles the fetch stalled for all icache misses. ",
+      "BriefDescription": "Counts the number of core cycles the fetch stalls because of an icache miss. This is a cumulative count of core cycles the fetch stalled for all icache misses. ",
```

Comment:
Looks good to me. Please increase the version in the header at the top of this file.